test.scss

```diff
@@ -0,0 +1,20 @@
+.testing {
+  font-size: 16px;
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets